profiling/tests/phpt/gc_collect_cycles_01.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+[profiling] test garbage collection being sampled for timeline
+--DESCRIPTION--
+This will check if for every garbage collection event a debug print will happen,
+indicating that gc events in the engine are sampled for the timeline feature.
+--SKIPIF--
+<?php
+if (!extension_loaded('datadog-profiling'))
+    echo "skip: test requires Datadog Continuous Profiler\n";
+ob_start();
+phpinfo(INFO_MODULES);
+$info = ob_get_clean();
+if (strpos($info, 'Experimental Timeline Enabled') === false)
+    echo "skip: datadog profiler is compiled without timeline support\n";
+?>
+--ENV--
+DD_PROFILING_ENABLED=yes
+DD_PROFILING_EXPERIMENTAL_CPU_TIME_ENABLED=no
+DD_PROFILING_EXPERIMENTAL_TIMELINE_ENABLED=yes
+DD_PROFILING_ALLOCATION_ENABLED=no
+DD_PROFILING_LOG_LEVEL=debug
+--FILE--
+<?php
+
+class SomeClass {
+    public SomeClass $someProperty;
```

Comment:
This is too new of a feature, needs to go back to PHP 7.1. It didn't fail in CI because this feature is not enabled for the phpt tests, so it gets skipped.